src/code/code_800F7260.c

```diff
@@ -33,19 +33,32 @@ char D_80133390[] = "SEQ H";
 char D_80133398[] = "    L";
 
 // bss
-extern SoundRequest sSoundRequests[0x100];
-extern SoundBankEntry D_8016BAD0[9];
-extern SoundBankEntry D_8016BC80[12];
-extern SoundBankEntry D_8016BEC0[22];
-extern SoundBankEntry D_8016C2E0[20];
-extern SoundBankEntry D_8016C6A0[8];
-extern SoundBankEntry D_8016C820[3];
-extern SoundBankEntry D_8016C8B0[5];
-extern u8 sSoundBankListEnd[7];
-extern u8 sSoundBankFreeListStart[7];
-extern u8 sSoundBankUnused[7];
-extern u8 sCurSfxPlayerChannelIdx;
-extern UnusedBankLerp sUnusedBankLerp[7];
+SoundBankEntry D_8016BAD0[9];
+SoundBankEntry D_8016BC80[12];
+SoundBankEntry D_8016BEC0[22];
+SoundBankEntry D_8016C2E0[20];
+SoundBankEntry D_8016C6A0[8];
+SoundBankEntry D_8016C820[3];
+SoundBankEntry D_8016C8B0[5];
+SoundRequest sSoundRequests[0x100];
+u8 sSoundBankListEnd[7];
+u8 sSoundBankFreeListStart[7];
+u8 sSoundBankUnused[7];
+ActiveSound gActiveSounds[7][3];
+u8 sCurSfxPlayerChannelIdx;
+u8 gSoundBankMuted[7];
+UnusedBankLerp sUnusedBankLerp[7];
+u16 gAudioSfxSwapSource[10];
+u16 gAudioSfxSwapTarget[10];
+u8 gAudioSfxSwapMode[10];
+Struct_8016E320 D_8016E320[4][5];
+u8 D_8016E348[4];
+u32 sAudioSeqCmds[0x100];
+unk_D_8016E750 D_8016E750[4];
+u8 D_8016F0E0[0xA0];
+AudioContext gAudioContext;
+void (*D_801755D0)(void);
+s32 padBSS[3];
```

Comment:
This looks like file padding. Is there more bss in this file afterwards which makes it necessary? If so, this probably indicates that it should be in a different file (tho we can decide to not split it if we want)

Reply:
this explains the large number of variables that are declared in this file then not used in the same file, right? its a separate data file where all this bss stuff lives

Reply:
When I originally tried putting it in the file I'd expect it to be in, I couldn't get a match. I don't think I played enough with these extra pads though to be fully confident. I'll mess around with it more

Reply:
yeah guess it depends on the order of stuff, havent looked closely.
it might just be the end of file boundary and you can just remove the array?

Reply:
Alright, so I can move the bss variables to a different file to reduce the number of externs, but I can't work out how to get rid of the extra pad bss yet